packages/api/amplify_api/README.md

```diff
@@ -13,76 +13,12 @@ For production use cases please use the latest, non-tagged versions of amplify-f
 | Category       | Android | iOS | Web | Windows | MacOS | Linux |
 | -------------- | :-----: | :-: | :-: | :-----: | :---: | :---: |
 | Analytics      |   🔴    | ✅  | 🔴  |   🔴    |  🔴   |  🔴   |
-| API (REST)     |   ✅    | ✅  | 🔴  |   🔴    |  🔴   |  🔴   |
-| API (GraphQL)  |   ✅    | ✅  | 🔴  |   🔴    |  🔴   |  🔴   |
+| API (REST)     |   ✅    | ✅  | ✅  |   ✅    |  ✅   |  ✅   |
+| API (GraphQL)  |   ✅    | ✅  | ✅  |   ✅    |  ✅   |  ✅   |
 | Authentication |   ✅    | ✅  | ✅  |   ✅    |  ✅   |  ✅   |
 | DataStore      |   ✅    | ✅  | 🔴  |   🔴    |  🔴   |  🔴   |
-| Storage        |   ✅    | ✅  | 🔴  |   🔴    |  🔴   |  🔴   |
+| Storage        |   ✅    | ✅  | ✅  |   ✅    |  ✅   |  ✅   |
 
 ## Getting Started
 
 ### Visit our [Web Site](https://docs.amplify.aws/) to learn more about AWS Amplify.
-
-## Changes for version 0.3.0 and above
```

Comment:
IMO this part has been here long enough that it doesn't need callout and not consistent with other categories, feel free to disagree and we can keep (although might be nice to have better documentation in these READMEs at some point).